snuba/admin/views.py

```diff
@@ -79,6 +79,9 @@ def clickhouse_queries() -> Response:
 def clickhouse_system_query() -> Response:
     req = request.get_json()
     try:
+        if req is None:
+            req = {}
```

Comment:
what made this necessary?

Reply:
also this will fail on the next line

Reply:
Added a comment, it's for a typing error. I thought it best to preserve the `KeyError` happening if `req` is `None` to avoid making logical changes.